articles/rock-paper-scissor-online/index.md

```diff
@@ -0,0 +1,260 @@
+- Front-end: To keep things simple, the front-end is built on basic HTML, CSS, JS with jQuery.
+- Back-end: An Express server that uses Socket.io.
+
+### Prerequisites
+ 
+- HTML is the standard markup language for web page. Here's a useful [guide](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML) for beginners.
+- CSS is style sheet language for improving the presentation of web pages. To learn more, check this [article](https://www.section.io/engineering-education/what-is-css/)
+- [BootStrap](https://getbootstrap.com/) is a popular HTML, CSS, JS library. An [article](https://getbootstrap.com/docs/4.0/getting-started/introduction/)  to get started.
+- [jQuery](https://jquery.com/) is a fast and feature-rich Javascript library. 
+- [Express.js](https://expressjs.com/) is a web application framework  for Node.js. Introductory article on Express.js: [link](https://www.section.io/engineering-education/express/).
+- [Socket.io](https://socket.io/) is a Real-Time Engine for Web. A helpful [article](https://www.section.io/engineering-education/understanding-socket/) to get started. This article covers the usage of Socket.io on both the server and client-side. These are essential to proceed further.
+
+
+
+### Starter Code
+
+In the interest of time and the intention to put more focus on Socket logic. Please download the starter code from this [Repository](https://github.com/HarishTeens/rps-online) on GitHub.  Follow the instructions as mentioned on README.md to set up the project on your local system.
+
+If you are only here for final version of the code, check out the [final branch](https://github.com/HarishTeens/rps-online/tree/final) of the same Repository.
+
+Here goes the file wise description of the Starter code,
+
+- ```public/index.html``` : A Home Page for the app with all the required UI Components. 
+- ```public/style.css``` : Styles for the entire application.
+- ```public/game.js``` : Client-Side JavaScript file to handle events.
+- ```app.js``` : Server Side NodeJS file for setting up the server.
+
+Note that the app is not functional yet, we need to write the necessary logic to make the app functional. Also, feel free to tweak the boiler code to match your style.
+
+There are two important files to focus on. One is  ```app.js``` that handles server-side logic. And the other is ```public/game.js``` that handles client-side logic.
+
+### Game Flow 
+
+![image](/engineering-education/nginx-reverse-proxy/rps.png)
+
+This is a Sequence diagram that shows the timeline of events. It's important to understand the flow before we start coding. So make sure you understand the diagram completely.
+
+Read through the below points if you are not familiar with Sequence Diagrams.
+
+- Time flows downward.
+- Arrows represent events. The start of Arrow denotes the Emitter, end of arrow denotes the listener. For example, Player1 emits the `Create Game` Event, Socket Server listens to this event.
```

Comment:
Join your sentences

**of an** arrow, **the** end

Event **and the** Socket server listens